panel/widgets/indicators.py

```diff
@@ -32,7 +35,7 @@
 from ..models import (
     HTML, Progress as _BkProgress, TrendIndicator as _BkTrendIndicator,
 )
-from ..pane.markup import Str
+from ..pane import Str
```

Comment:
Looks like `Str` actually needs to be exported in `panel/pane/__init__.py`. Not sure why it's not.